### PR TITLE
Use main instead of master

### PR DESCRIPTION
**Pull request by kinow**
Hi @afs 

I'm using Jena's site as reference while migrating SIS' site from old SVN to a new git site. Doing so, I'm also reviewing our docs.

Found `master` on README and hugo scripts in the root directory. Then created a branch for that, and then search/replace'd the rest in source. They are in separate commits, so happy to drop a commit if necessary :+1: 

Cheers
Bruno

### CODE REVIEW

**Comment by kinow** [comment]
I did not change two occurrences of master:

- `README.md` where it says "the jenkins (master pub key) credentials."
- `fuseki-layout.md` where it says "The master copy is kept in the system database."

As I wasn't sure what would be the replacements, nor if they would be necessary (but happy to push a new commit fixing these)

Bruno

File: build-script [hunk 1]
```diff
@@ -6,7 +6,7 @@
 ## -------------------
 
 # Specify source and site branches
-SOURCE="master"
+SOURCE="main"
```
Comment (afs) [inline build-script:9]: I think `build-script` isn't used any more. IIRC it was the original way to build the site but was replaced by `Jenkinsfile`, and that does not call `build-script`. `Jenkinsfile` has similar shell fragments.

Only way to find out is delete `build-script` and see!

Can make this change and separately investigate the effect of `build-script`.

**Review by afs — APPROVED** [review]

File: build-script (hunk 1 above)
Comment (afs) [reply build-script:9]: There is no way `build-script` is active. It checks out the old default branch by name.

Comment (kinow) [reply build-script]: Pushed a commit removing the `build-script` file. If no lights blink, and there's no smoke coming out anywhere, it should be safe to leave it out :laughing: but I agree it's probably not used. I thought it was used to build something locally only.

Comment (afs) [reply build-script]: Hazy memory but I think the first setup did not use the `Jenkinsfile` feature on Jenkins; instead it was a bash script and `build-script` is copy for reference.

**Comment by kinow** [comment]
Fixed conflicts, merging :+1:

**Pull request merged**